aggregator/handler/router/traffic_control_options.go

```diff
@@ -84,9 +64,10 @@ func (t *TrafficControllerType) UnmarshalYAML(unmarshal func(interface{}) error)
 
 // TrafficControllerConfiguration configures the traffic controller.
 type TrafficControllerConfiguration struct {
-	Type        TrafficControllerType `yaml:"type"`
-	RuntimeKey  string                `yaml:"runtimeKey" validate:"nonzero"`
-	InitTimeout *time.Duration        `yaml:"initTimeout"`
+	Type         TrafficControllerType `yaml:"type"`
+	DefaultValue bool                  `yaml:"defaultValue"`
```

Comment:
Let's make this a `*bool` so you don't accidentally set this to false in the options when you forget to set it in the config.

Reply:
sg